GPU/D3D11/FramebufferManagerD3D11.cpp

```diff
@@ -458,8 +454,14 @@ void FramebufferManagerD3D11::BindFramebufferColor(int stage, VirtualFramebuffer
 		} else {
 			draw_->BindFramebufferAsTexture(framebuffer->fbo, stage, Draw::FB_COLOR_BIT, 0);
 		}
-	} else {
+	} else if (framebuffer != currentRenderVfb_) {
 		draw_->BindFramebufferAsTexture(framebuffer->fbo, stage, Draw::FB_COLOR_BIT, 0);
+	} else {
+		// Badness on D3D11 to bind the currently rendered-to framebuffer as a texture.
+		ID3D11ShaderResourceView *view = nullptr;
```

Comment:
Probably not, just wanted to make sure all cases were handled. Should probably report.